src/map/battle.cpp

```diff
@@ -929,7 +929,7 @@ int battle_calc_cardfix(int attack_type, struct block_list *src, struct block_li
 				}
 				if( flag&BF_SHORT )
 					cardfix = cardfix * (100 - tsd->bonus.near_attack_def_rate) / 100;
-				else	// BF_LONG (there's no other choice)
+				else if (battle_getcurrentskill(src) != SR_GATEOFHELL)	// BF_LONG (there's no other choice)
```

Comment:
I don't like the notion of skill_check in here, why do we need this ?
If anything I'd rather have a BF_IGNORE_LONG, but honestly I don't really get it.